main.go

```diff
@@ -0,0 +1,105 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"reflect"
+	"runtime"
+	"strconv"
+
+	"github.com/getsentry/raven-go"
+	"github.com/iotaledger/giota"
+	"github.com/joho/godotenv"
+)
+
+type indexRequest struct {
+	Trytes []giota.Trytes `json:"trytes"`
+}
+
+func main() {
+	raven.CapturePanic(func() {
+
+		// Load ENV variables
+		err := godotenv.Load()
+		if err != nil {
+			log.Fatal("Error loading .env file")
+		}
+
+		// Attach handlers
+		http.HandleFunc("/", indexHandler)
+		http.HandleFunc("/pow", powHandler)
+
+		// Fetch port from ENV
+		port := os.Getenv("PORT")
+		fmt.Print("Listing on http://localhost:" + port)
+
+		// Start listening
+		http.ListenAndServe(":"+port, nil)
+
+	}, nil)
+}
+
+func indexHandler(w http.ResponseWriter, r *http.Request) {
+	fmt.Print("\nProcessing trytes\n")
+	if r.Method == "POST" {
+
+		// Unmarshal JSON
+		b, err := ioutil.ReadAll(r.Body)
+		if err != nil {
+			http.Error(w, "Invalid request method", http.StatusBadRequest)
+			return
+		}
+		defer r.Body.Close()
+		req := indexRequest{}
+		json.Unmarshal(b, &req)
+
+		// Convert []Trytes to []Transaction
+		txs := make([]giota.Transaction, len(req.Trytes))
+		for i, t := range req.Trytes {
+			tx, _ := giota.NewTransaction(t)
+			txs[i] = *tx
+		}
+
+		// Get configuration.
+		provider := os.Getenv("PROVIDER")
+		minDepth, _ := strconv.ParseInt(os.Getenv("MIN_DEPTH"), 10, 64)
+		minWeightMag, _ := strconv.ParseInt(os.Getenv("MIN_WEIGHT_MAGNITUDE"), 10, 64)
+
+		// Async sendTrytes
+		api := giota.NewAPI(provider, nil)
+		_, pow := giota.GetBestPoW()
```

Comment:
I don't know much about Go.  What does that '_, ' do?

Reply:
Signals to the compiler that the variable is unused. In this case, the first return value is an error, which we just ignore.